SocialCareCaseViewerApi.Tests/V1/Gateways/DatabaseGateway.Tests.cs

```diff
@@ -1082,8 +1083,8 @@ public void GetWarningNoteByIdReturnsReturnsMultipleReviewsWithTheWarningNote()
         {
             var warningNote = TestHelpers.CreateWarningNote();
             var firstReview = TestHelpers.CreateWarningNoteReview(warningNote.Id);
-            var secondReview = TestHelpers.CreateWarningNoteReview(warningNote.Id);
-            var thirdReview = TestHelpers.CreateWarningNoteReview(warningNote.Id);
+            var secondReview = TestHelpers.CreateWarningNoteReview(warningNote.Id, true);
+            var thirdReview = TestHelpers.CreateWarningNoteReview(warningNote.Id, false);
```

Comment:
For readability, could we used named arguments here? I think that's the right syntax... :sweat_smile:

```suggestion
            var secondReview = TestHelpers.CreateWarningNoteReview(warningNote.Id, disclosedWithIndividual: true);
            var thirdReview = TestHelpers.CreateWarningNoteReview(warningNote.Id, disclosedWithIndividual: false);
```